### PR TITLE
[SD-3014] Modify list of fields applied with templates

### PR DESCRIPTION
**Pull request by marwoodandrew**
This list of fields applied when creating the template has been unified with the list that is used when creating the template.

### CODE REVIEW

File: client/app/scripts/superdesk-templates/templates.js [hunk 1]
```diff
@@ -21,6 +21,33 @@
...
+                'pubstatus',
+                'description',
+                'body_html',
+                'body_text',
+                'place',
+                'located',
+                'creditline',
+                'ednote',
+                'language',
+                'place'
```
Comment (sivakuna-aap) [inline client/app/scripts/superdesk-templates/templates.js]: Duplicate

**Comment by sivakuna-aap** [comment]
:+1:

File: client/app/scripts/superdesk-templates/templates.js (hunk 1 above)
Comment (marwoodandrew) [inline client/app/scripts/superdesk-templates/templates.js]: Well spotted

**Comment by akintolga** [comment]
:+1:

**Comment by plamut** [comment]
:+1:

**Pull request merged**